### DISCUSSION TITLE
Deprem Bağış Bilgilendirme ve Dolandırıcılığa Karşı Uyarı Sitesi

### DISCUSSION THREAD

**Discussion by atakanatamert** [acikkaynak/deprem-yardim-projesi#70] [Show and tell]
İnsanları doğru bağış kanallarına yönlendirmek ve yaygın olarak görülen dolandırıcılık yöntemlerine karşı bilgilendirmek amacıyla Next.js + Chakra UI kullanarak static bir site oluşturdum. Fakat legalitesinden emin olamadığım için deploylamadım. Aceleyle yaptığım için kod kalitesi ve içerik maalesef yeterli değil, fakat bu haliyle belki template olarak kullanılabilir. Veya ileride geliştirilerek (Kullanıcıların yeni dolandırıcılık uyarılarını eklemeleri vb. ) bir fikir olarak başka bir projeye eklenebilir. 

Proje reposuna [buradan](https://github.com/atakanatamert/deprem-bagis-bilgilendirme) ulaşabilirsiniz. 

<p float="left">
<img src="https://user-images.githubusercontent.com/32536245/218269905-4c905627-5628-4090-b425-ba88d5282019.png"  width="45%" >
<img src="https://user-images.githubusercontent.com/32536245/218269925-4bede348-84b8-43fe-991d-0c08c62897b7.png"  width="45%" >
</p>

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz siber güvenlik ekibimiz tarafından sürekli olarak kontrol edilerek ilgili aksiyonlar alınmaktadır.

**Comment by sercangoger** [comment]
bu site de 
sosyal medya linkini ihbar et
site ihbar et
gibi iki  input oluşturulabilir. bu sayede hızlı bir şekilde herkes tarafından dolandırııcılık bilgilendirmesi yapılabilir. bu sayede 
siber güvenlik ekiplerine kaynak sağlanabilecektir.
